.claude/database-ops/summary.sh

```
@@ -0,0 +1,194 @@
+
+if [[ ! -f "$ENV_FILE" ]]; then
+  echo "ERROR: .env not found at $ENV_FILE" >&2
+  exit 1
+fi
+# shellcheck disable=SC1090
+source "$ENV_FILE"
+
+# ── Date handling ──────────────────────────────────────────────────────────────
+TZ_NAME="${3:-Asia/Kolkata}"
+IST_DATE="${1:-}"
+OFFSET_SECS=19800  # IST = UTC+5:30 = 19800s
```

Shellcheck (0.11.0)

[warning] OFFSET_SECS appears unused. Verify use (or export if used externally).

(SC2034)